src/simple-cache-client.ts

```diff
@@ -165,6 +166,32 @@ export class SimpleCacheClient {
     return await client.setFetch(cacheName, setName);
   }
 
+  /**
+   * Add an element to a set in the cache.
+   *
+   * After this operation, the set will contain the union
+   * of the element passed in and the elements of the set.
+   * @param {string} cacheName - Name of the cache to store the set in.
+   * @param {string} setName - The set to add elements to.
+   * @param {(string | Uint8Array)} element - The data to add to the set.
+   * @param {CollectionTtl} [ttl] - TTL for the set in cache. This TTL takes precedence over the TTL used when initializing a cache client. Defaults to client TTL.
+   */
+  public async setAddElement(
+    cacheName: string,
+    setName: string,
+    element: string | Uint8Array,
+    ttl?: CollectionTtl
+  ): Promise<CacheSetAddElement.Response> {
+    return (
+      await this.setAddElements(
+        cacheName,
+        setName,
+        [element] as string[] | Uint8Array[],
```

Comment:
LMK if someone can figure out how to get rid of this ugly cast as string[] | Uint8Array[]

Reply:
you could just define the var as an explicitly typed const first:

```
const elements: string[] | Uint8Array[] = [element];
```

Reply:
The compiler won't allow that... unless I do the same ugly cast:
```
const elements: string[] | Uint8Array[] = [element] as string[] | Uint8Array[]
```

Reply:
If it makes you feel better, I ran into the same problem and did something even worse.

```
      if (typeof props.value === 'string') {
        return Momento.listConcatenateFront(
          props.cacheName,
          props.listName,
          [props.value],
          props.ttl,
          props.truncateBackToSize
        );
      } else {
        return Momento.listConcatenateFront(
          props.cacheName,
          props.listName,
          [props.value],
          props.ttl,
          props.truncateBackToSize
        );
      }
```